contracts/interfaces/modules/royalty/IRoyaltyModule.sol

```diff
@@ -40,6 +40,27 @@ interface IRoyaltyModule is IModule {
     /// @param accumulatedRoyaltyPoliciesLimit The maximum number of accumulated royalty policies an IP asset can have
     event IpGraphLimitsUpdated(uint256 maxParents, uint256 maxAncestors, uint256 accumulatedRoyaltyPoliciesLimit);
 
+    /// @notice Event emitted when a license is minted
+    /// @param ipId The ipId whose license is being minted (licensor)
+    /// @param royaltyPolicy The royalty policy address of the license being minted
+    /// @param licensePercent The license percentage of the license being minted
+    /// @param externalData The external data custom to the royalty policy being minted
+    event LicenseMinted(address ipId, address royaltyPolicy, uint32 licensePercent, bytes externalData);
```

Comment:
The reason it was added there was because for the offchain db it required an event that returned the royaltyPolicy address and the licensePercent (along with external data if we need it in the future)

`LicenseTokensMinted` doesn't return these (at least directly as this info may be contained inside the license terms id).

We can also rename - how would you phrase it?

Reply:
Might be name like "LicensedWithRoyalty"?

Reply:
renamed in https://github.com/storyprotocol/protocol-core-v1/pull/215/commits/f4290bad1374e718d84e1ef2272eb0487a494f38